lua/nvim-tree/lib.lua

```diff
@@ -113,12 +113,17 @@ end
 
 function M.open(cwd)
   M.set_target_win()
+  print("open x")
   if not core.get_explorer() or cwd then
+  print("no explorer")
     core.init(cwd or vim.loop.cwd())
   end
   if should_hijack_current_buf() then
+    print("should hijack")
```

Comment:
You can use logs e.g. `log.line("dev", "SessionLoadPost")` for easier debugging.

Set
```lua
	  log = {
		  enable = true,
		  truncate = true,
		  types = {
			  dev = true,
		  },
	  },
```

and tail `~/.cache/nvim/nvim-tree.log`